packages/phrase/src/pull-translations.test.ts

```diff
@@ -77,7 +77,7 @@ describe('pull translations', () => {
     it('should resolve', async () => {
       await expect(runPhrase(options)).resolves.toBeUndefined();
 
-      expect(jest.mocked(writeFile)).toHaveBeenCalledTimes(2);
+      expect(jest.mocked(writeFile)).toHaveBeenCalledTimes(4);
```

Comment:
Strange that this is 4 instead of 3 right? I Looks like there's an empty write on line 100 below.

Reply:
I think this is because its trying to write for the languages in the config.

I checked the code and that is what I assumed to be the case. We can investigate this further.